onnxruntime/core/flatbuffers/flatbuffers_utils.h

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <core/common/status.h>
+#include <core/graph/basic_types.h>
+#include <core/session/onnxruntime_c_api.h>
+
```

Comment:
the C API is a really high level thing. why does a low level thing like this library have a dependency on it?

Reply:
Oh, this include is only for the "ORT_MUST_USE_RESULT" which is defined in the c_api

---
In reply to: [494752637](https://github.com/microsoft/onnxruntime/pull/5276#discussion_r494752637) [](ancestors = 494752637)